### ISSUE TITLE
reCAPTCHA Enterprise API

### ISSUE THREAD

**Issue by emadbatwa** [chvarkov/google-recaptcha#70]
Hello @chvarkov .
are you support reCAPTCHA Enterprise API ?
i get 500 response
```json
{
    "statusCode": 500,
    "message": "Invalid module configuration. Please check public-secret keys."
}
```

**Comment by chvarkov** [comment]
Hi @emadbatwa, thanks for submit issue.
Not supported yet, I started working on this.

**Comment by chvarkov** [comment]
Published and available since v3.1.0

**Comment by emadbatwa** [comment]
great ! thank you @chvarkov
